packages/bot-web-ui/src/components/load-modal/__tests__/recent.spec.tsx

```diff
@@ -53,4 +54,21 @@ describe('Recent component of load modal', () => {
         expect(strategy_previw).toBeInTheDocument();
         expect(container).toBeInTheDocument();
     });
+
+    it("on click of Why can't I see my recent bots the accordion should close and open", () => {
```

Comment:
nitpick to the name, it would be better to start with "should [expected result/action] ... and so on